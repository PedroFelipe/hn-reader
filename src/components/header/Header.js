import React, { Component } from 'react'

import './header.css'

class Header extends Component {
  render() {
    return (
      <header className="section header">
        <div className="container">
          <h1 className="title is-4 header-title">
            <span className="header-logo">Y</span>
            Hacker News Reader
          </h1>
        </div>
      </header>
    )
  }
}

export default Header
