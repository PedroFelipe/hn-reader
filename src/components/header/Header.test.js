import React from 'react'
import renderer from 'react-test-renderer'
import { MemoryRouter as Router, withRouter } from 'react-router-dom'

import Header from './Header'

test('Header renders correctly', () => {
  const tree = renderer.create(
    <Router>
      <Header />
    </Router>
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
