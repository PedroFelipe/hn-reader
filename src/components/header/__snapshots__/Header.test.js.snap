// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header renders correctly 1`] = `
<header>
  <div
    className="section header"
  >
    <div
      className="container"
    >
      <h1
        className="title is-4 header-title"
      >
        <span
          className="header-logo"
        >
          Y
        </span>
        Hacker News Reader
      </h1>
    </div>
  </div>
  <nav
    className="section header-nav"
  >
    <div
      className="container"
    >
      <ul
        className="header-nav-list"
      >
        <li
          className="header-nav-item"
        >
          <a
            aria-current="true"
            className="header-nav-url header-nav-url-active"
            href="/"
            onClick={[Function]}
            style={Object {}}
          >
            top
          </a>
        </li>
        <li
          className="header-nav-item"
        >
          <a
            aria-current={false}
            className="header-nav-url"
            href="new"
            onClick={[Function]}
            style={undefined}
          >
            newest
          </a>
        </li>
        <li
          className="header-nav-item"
        >
          <a
            aria-current={false}
            className="header-nav-url"
            href="best"
            onClick={[Function]}
            style={undefined}
          >
            best
          </a>
        </li>
        <li
          className="header-nav-item"
        >
          <a
            aria-current={false}
            className="header-nav-url"
            href="ask"
            onClick={[Function]}
            style={undefined}
          >
            ask
          </a>
        </li>
        <li
          className="header-nav-item"
        >
          <a
            aria-current={false}
            className="header-nav-url"
            href="show"
            onClick={[Function]}
            style={undefined}
          >
            show
          </a>
        </li>
        <li
          className="header-nav-item"
        >
          <a
            aria-current={false}
            className="header-nav-url"
            href="jobs"
            onClick={[Function]}
            style={undefined}
          >
            jobs
          </a>
        </li>
      </ul>
    </div>
  </nav>
</header>
`;
